app.yaml.base

```diff
@@ -38,15 +38,10 @@ handlers:
           static_files: static/\1
           upload: static/(.*\.(bmp|gif|ico|jpeg|jpg|png))
 
-# site root
-        - url: /
-          static_files: static/index.html
-          upload: static/index.html
-          expiration: "15m"
-
 # All URLs for Back End logic
-        - url: /api/.*
+        - url: /.*
```

Comment:
But still it's going to be served via "script: main.app" in both cases. Is it not going to be the same?